pdns/recursordist/Makefile.am

```diff
@@ -85,7 +86,7 @@ pdns_recursor_SOURCES = \
 	misc.hh misc.cc \
 	mplexer.hh \
 	mtasker.hh \
-	mtasker_context.hh mtasker_fcontext.cc \
+	mtasker_context.hh mtasker_context.cc \
```

Comment:
Not wanting to be a PITA, but could you swap these, so source file first, then header?